desk.acc/this.apple.s

```diff
@@ -251,6 +251,9 @@ str_iie:
 str_iie_enhanced:
         PASCAL_STRING "Apple IIe (enhanced)"
 
+str_iie_card:
+        PASCAL_STRING "Apple IIe Card"
```

Comment:
But what shall we use for a bitmap? ;-)

(Seriously... I can do up a bitmap of the card easily enough...)

Reply:
I was just going to take some "inspiration" from the IIe Card app. :-)

Reply:
I made a bitmap, but it causes the end of the DA to go past $13FF...  is that a hard limit?

Reply:
Ahh it looks like we can go up to $1CFF unless I am mistaken.

Reply:
It came out better than I expected for a first try drawing in 1s and 0s in my text editor...

![card_bmap](https://user-images.githubusercontent.com/17011595/38439734-a82c1028-3993-11e8-9d03-8d6a1993d2fc.png)

Reply:
If README.md is still correct, the window/icon tables start at $1B00 so $1AFF would be the hard limit in AUX. 

(The DA right now replicates itself between main and aux for easy bank switching, but that doesn't need to be maintained.)

Reply:
(and the current $13FF was a copy/paste from other DAs that needed to keep space free for file bufffers, totally arbitrary)

Reply:
Funny, I read that right and changed my .assert bounds check to $1B00, but somehow wrote $1CFF in my comment.  Brains are weird.

I'll see what the icon looks like with 2px verticals and some other tweaks.